third_party/ijar/BUILD

```diff
@@ -97,3 +97,17 @@ filegroup(
     srcs = glob(["**"]),
     visibility = ["//visibility:public"],
 )
+
+filegroup(
+    name = "embedded_java_tools",
+    srcs = glob(
+        ["**"],
+        exclude = ["BUILD"],
+    ),
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
```

Comment:
nit: Why do you need this filegroup? Is it the naming? Use alias instead maybe?

Reply:
Thanks for spotting this. It looks like this was a leftover target. I removed it.